lib/strategies/default.js

```diff
@@ -0,0 +1,145 @@
+'use strict';
+
+var keyCannotMutateValue = require('../key-cannot-mutate-value');
+var getSelection = require('../input-selection').get;
+var setSelection = require('../input-selection').set;
+var isBackspace = require('../is-backspace');
+var isDelete = require('../is-delete');
+var Formatter = require('../formatter');
+
+function DefaultStrategy(options) {
```

Comment:
@intelliot and I talked about this and we think `BaseStrategy` is a slightly better name for this, because it's a base class. Thoughts?